src/factories/contractFactory.ts

```diff
@@ -267,18 +268,36 @@ export default class ContractFactory {
     assert.isETHAddressHex('address', address);
     const contract = new PercentageTransferManagerContract(address, this.provider, this.contractDefaults);
     this.abiArray.forEach(
-      (abi): void => {
-        contract.addABItoDecoder(abi);
-      },
+        (abi): void => {
+          contract.addABItoDecoder(abi);
+        },
     );
     return contract;
   }
 
+
   public async getLockUpTransferManagerContract(address: string): Promise<LockUpTransferManagerContract> {
     assert.isETHAddressHex('address', address);
-    return new LockUpTransferManagerContract(address, this.provider, this.contractDefaults);
+    const contract = new LockUpTransferManagerContract(address, this.provider, this.contractDefaults);
```

Comment:
Could you do this same change in this file's `getVestingEscrowWalletContract`? We missed it in the last PR